src/page/MoviePage.jsx

```diff
@@ -58,10 +70,13 @@ export default class MoviePage extends React.Component {
 
   loadMovies(page) {
     const { type, getActionDispatcher } = this.props;
-    return getMovies(type, page)
+    getActionDispatcher({
+      type: MOVIES.REQUEST,
+    })();
+    return getMovies({ type, page })
       .then(({ movies, totalResults }) => {
         getActionDispatcher({
```

Comment:
Don't create actions here. This is antipattern. Use action creators please.
https://redux.js.org/basics/actions